Include/internal/pycore_symtable.h

```diff
@@ -58,6 +58,7 @@ typedef struct {
                .end_col_offset = (n)->end_col_offset }
 
 static const _Py_SourceLocation NO_LOCATION = {-1, -1, -1, -1};
+static const _Py_SourceLocation NEXT_LOCATION = {INT_MAX, INT_MAX, INT_MAX, INT_MAX};
```

Comment:
This is a good idea.

We probably need to change remove_redundant_nops to treat this ad no location. Maybe it should be (-2, -2, -2, -2) and then just compare to 0?

Reply:
There are some checks for  `NO_LOCATION` that compare to 0, but it isn't clear to me which should support `NEXT_LOCATION` and which shouldn't, so I'm a bit reluctant to do that.

Reply:
With {2, -2, -2, -2}, `basicblock_remove_redundant_nops` and `propagate_line_numbers` needed changing.
`basicblock_remove_redundant_nops` needs to treat NEXT_LOCATION like NO_LOCATION
`propagate_line_numbers` needs to treat NEXT_LOCATION  *not* like NO_LOCATION